chatterbot/adapters/logic/mixins.py

```diff
@@ -1,4 +1,9 @@
 class TieBreaking(object):
+    """
+    TieBreaking determines which adapter's result should be used
+    in the event that mutliple adapters return the same highest
+    confidence value.
```

Comment:
Tie breaking isn't actually used in for tie breaking between multiple adapters. If two adapters return the same confidence value, then the adapter that is listed first will take priority. The TieBreaking class is used in a single adapter when there are multiple known statements that are exact matches for a given input statement.